src/Components/DnDLayout/GridLayout.tsx

```diff
@@ -119,6 +124,14 @@ const GridLayout = ({ isLayoutLocked = false }: { isLayoutLocked?: boolean }) =>
     [isLayoutLocked, layout]
   );
 
+  useEffect(() => {
+    const updatedWidgetClasses = layout.reduce<WidgetClasses>((acc, curr) => {
+      acc[curr.i] = `widget-columns-${curr.w}`;
+      return acc;
+    }, {});
+    setWidgetClasses(updatedWidgetClasses);
+  }, [layout]);
```

Comment:
@CodyWMitchell I'd be curious how often this gets fired as the user interacts with the page  - and if we have a way of reducing the amount of times the effect is called. We really only care when the number of columns changes right? I wonder if we should listen for when the `layoutVariant` changes instead, then iterate over the layout to update the classes 🤔